solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -8,6 +8,7 @@ import "../../ConverterVersion.sol";
 import "../../interfaces/IConverter.sol";
 import "../../interfaces/IConverterAnchor.sol";
 import "../../interfaces/IConverterUpgrader.sol";
+import "../../../INetworkSettings.sol";
```

Comment:
IMHO, you should also update `ConverterVersion`, so that it'd be easier for us to keep track on opted-in/upgraded converters with this feature

Reply:
Done.